lib/ffprober/ffmpeg/finder.rb

```diff
@@ -0,0 +1,23 @@
+module Ffprober
+  module Ffmpeg
+    class Finder
+      def self.path
+        @path ||= begin
+          path = ENV['PATH'].split(File::PATH_SEPARATOR).find do |path_to_check|
```

Comment:
Prefer `detect` over `find`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.